chainer/functions/connection/convolution_2d.py

```diff
@@ -38,6 +41,15 @@ def __init__(self, stride=1, pad=0, use_cudnn=True, cover_all=False,
         self.cover_all = cover_all
         self.deterministic = deterministic
 
+        self.fp16_compute_mode = getattr(function._thread_local,
+                                         'fp16_compute_mode', False)
+        if self.fp16_compute_mode:
+            # fp16 compute is available on GPUs with compute capability 6.0.
+            if not _cc == '60':
```

Comment:
It cannot be decided if a GPU has full FP16 capability from simple large/small relation of the compute capability number. Please refer to [this link](http://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html#arithmetic-instructions) about what GPU/compute capability has full FP16 capability.